src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import { Billing } from "./components/Billing";
+import { Billing } from "./components/Billing"; // index reexport for handling import mess
```

Comment:
ezt nem ertem, csinaljak egy indexek a componenseknel es onnan egy importban arra erted?

Reply:
csinalhatsz egy index filet a compnoents-be ami reexportal mindent amit hasznalsz ebbol a folder sub-treebol (ebben az esetben a komponenseket. Akkor lehet egy olyat hogy:

`import { CompA, CompB, ... } from './components;` 

Tell tale sign erre hogy ha importkor drilldown-olnod kell, mint feljebb:

`./components/Billing` vagy `../valami/sub-valami`

Es ezt megkoszonod magadnak a kesobbiekben.